build.gradle

```diff
@@ -40,6 +40,9 @@ subprojects {
         maven {
             url "https://repository.apache.org/snapshots"
         }
+        maven {
```

Comment:
It doesn't seem to be a big deal, but why are we adding sonatype snapshots in this PR? It is not clear how it is related.

Reply:
Note that this is reverting a PR, thus re-adding the sonatype snapshot repo, which the previous PR removed because it was dead code.

Reply:
I think you are saying that we don't need this addition as it was dead gradle configuration removed by the original PR. At this point, I can go either way, removing it here or in a separate PR.

Reply:
@RaulGracia what's your take?

Reply:
@EronWright is right, I was just reverting all the changes literally. I can keep that change that was removing that repo, not a problem.